WEB/Scraper/Scraper.py

```diff
@@ -0,0 +1,10 @@
+from twitterscraper import query_tweets
+import datetime as dt
+
+if __name__ == '__main__':
+    data = query_tweets("state of the union", limit=10, begindate=dt.date(2018, 12, 20)) #limit is 10 tweets here
+
+with open("data.txt", mode='w') as file: # Use file to refer to the file object
+    for tweet in data[:10]: #at least 10 results within the minimal possible time/number of requests
+        print(tweet.text.encode('utf-8'))
```

Comment:
We should collect the data for a tweet into a serializable data structure and save it as json data. (https://docs.python.org/3/library/json.html)

For example, this snippet would store all of the data that is provided by the twitterscraper library

```
        serializable_tweet = {}
        serializable_tweet['user'] = tweet.user
        serializable_tweet['fullname'] = tweet.fullname
        serializable_tweet['id'] = tweet.id
        serializable_tweet['url'] = tweet.url
        serializable_tweet['timestamp'] = tweet.timestamp.isoformat()
        serializable_tweet['text'] = tweet.text
        serializable_tweet['replies'] = tweet.replies
        serializable_tweet['retweets'] = tweet.retweets
        serializable_tweet['likes'] = tweet.likes
        serializable_tweet['html'] = tweet.html
        print(json.dumps(serializable_tweet))
        file.write(json.dumps(serializable_tweet) +'\n')
```